cpp/oai_pmh_harvester.cc

```diff
@@ -241,11 +311,27 @@ int Main(int argc, char **argv) {
     }
 
     if (argc != 6 and argc != 7)
-        Usage();
+        ::Usage("[--skip-dups] [--ignore-ssl-certificates] base_url metadata_prefix [harvest_set_or_identifier] "
```

Comment:
Wouldn't it be a good idea to use a separate dups_db for each data source? (even if the hashes do not overlap, maybe for performance reasons? Or to be able to reset the hashes for a single source?)

In that case it would be necessary to pass the path to the dups_db via argv

Reply:
Done.